pom.xml

```diff
@@ -97,7 +97,19 @@
 		</dependency>
 	</dependencies>
 
-	<build>
+    <!-- Allow finding fixed moditect version -->

```

Comment:
Could you please extend this comment in a way that it somehow says when the snapshot repository can be removed again? Something like: "Remove this when we have an official moditect release containing bugfix X". Or "Remove this when modditect version x is released"?